html/user-profile.html

```diff
@@ -5,35 +5,56 @@
     <meta content="text/html; charset=UTF-8">
     <link rel="stylesheet" type="text/css" href="../css/style.css" />
 </head>
-<body>
-  <header>
-    <nav>
-      <ul>
-        <li><a href="user-profile.html">User Profile</a></li>
-        <li><a href="user-list.html">User List</a></li>
-        <li><a href="create-user.html">Create User</a></li>
-        <li><a href="edit-user.html">Edit User</a></li>
-      </ul>
-    </nav>
-  </header>
-  <table> 
-      <h3>User Profile</h3>
-    <tr>
-      <td>ID:</td>
-      <td>#0000</td>
-    </tr>
-    <tr>
-      <td>Name:</td>
-      <td>John Smith</td>
-    </tr>
-    <tr>
-      <td>Phone:</td>
-      <td>000-000-0000</td>
-    </tr>
-    <tr>
-      <td>Email:</td>
-      <td>john.smith@banno.com</td>
-    </tr>
-  </table>
-</body>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="user-profile.html">User Profile</a></li>
+          <li><a href="user-list.html">User List</a></li>
+          <li><a href="create-user.html">Create User</a></li>
+          <li><a href="edit-user.html">Edit User</a></li>
+        </ul>
+      </nav>
+    </header>
+    <h3>User Profile</h3>
+    <table> 
+      <tr>
+        <td>ID:</td>
+        <td>#0000</td>
+      </tr>
+      <tr>
+        <td>Name:</td>
+        <td>John Smith</td>
+      </tr>
+      <tr class="expand">
+        <td>Phone:</td>
+        <td>000-576-5309</td>
+      </tr>
+      <tr class="expand">
+        <td>Email:</td>
+        <td>john.smith@banno.com</td>
+      </tr>
+      <tr>
+        <td>
+          <button id="bText"  Onclick="openClose()">Collapse</button>
+        </td>
+      </tr>
+    </table>
+    <script>
+      function openClose() {
+        var collapse = document.getElementsByClassName("expand");
```

Comment:
If you are only going to be collapsing/expanding a single card on this page, then you could just use `document.getElementById/getElementByClassName` rather than `document.getElementsById/getElementsByClassname`.

If you are intending to get multiple elements by classname, you are probably going to run into some problems that I'll mention in the next comment.

Reply:
Yeah the intent is to access to just the one element at a time at the moment, but I'd like to figure out how to be able to make it work separately for each card in the list.